Fix/moder

whaeck opened the pull request:
Little update to be able to convert MF28 to binary files. This addresses issue #162

the feature/pn-iaea branch needs to be merged before this one.

nathangibson14 commented on tests/59/input:
```diff
@@ -0,0 +1,14 @@
+moder
+-30 40
+moder
+-31 41
```
There's no check that tape40 and tape41 are actually what they should be.  If these lines are going to stay in the test suite, they should be used.

Running this test myself, I see differences in significant digits (e.g., switch from floating point to exponential) in tape20 and tape40.  Tape21 and tape41 (what's of interest here, of course!) look closer, with the changes looking like switching format but retaining significant digits.  I think that's expected NJOY behavior, so this looks fine.

What are your thoughts?

whaeck replied:
That's the issue with some ENDF files. When using MODER, the file gets reformatted into they NJOY writes its own ENDF files. ENDFtk would essentially lead to the same thing: the ENDF file looks different but it contains the same information. The only way to be sure is to use properly formatted files in the first place which should be reproduced exactly.

I'll replace the original tape20 and tape21 with the reformatted tape40 and tape41. The ACE file might change a bit tough - due to roundoff.

nathangibson14 reviewed (commented):
Left a question on the test case.

This is good to go with a relaxed approach to testing.  But we should make sure we're happy with that relaxed approach.

nathangibson14 reviewed (approved):
As I did with the other PR I looked at this morning, I'll approve this now.  If you change the test, I give my blessing to you approving it yourself.  And if you don't, well, it's already approved.

whaeck commented:
I replaced the input tapes with properly formatted ones. As a result, the ACE files for test 48 and the new test 59 no longer agree within a relative error of 1e-9. They do agree with a relative error of 1e-6, which is essentially the precision in the ENDF file. Tapes 40 and 41 are now also compared (since they should be exactly identical to the input tapes, I modified the CMakeLists.txt file to copy the input files twice instead of creating reference tapes in the test case itself).
